lib/raven/configuration.rb

```diff
@@ -127,6 +127,12 @@ def initialize
       self.sanitize_fields = []
       self.sanitize_credit_cards = true
       self.environments = []
+
+      self.release = ENV['HEROKU_SLUG_COMMIT']
+
+      if self.release.nil? || self.release.empty?
+        self.release = `git rev-parse --short HEAD`.strip
```

Comment:
If the app isn't versioned with git, ``git rev-parse --short HEAD`.strip`` will gracefully fallback to an empty string. I can go ahead and remove this, but keeping it in place might be a great added feature for devs with no harmful failure case. Thoughts?